IanGRap/Squish
# Put new bomb icon on bomb block

Contributors: Coulter

Goal: Replace stock bomb icon with one made by the team.

Files Changed:
- Cross Bomb Block prefab: replaced the bomb icon on the front.

How to test:
- Start the match scene. Either wait for a bomb to spawn or put one in yourself. 
- Observe the new icon.
- Judge the new icon.
